applications/e-commerce/soccer.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en"> 
+    <head>
+        <meta charset="utf-8"> <title>Soccer</title>     
+    </head>
+
+<body>
+    <header>
+        <h1>Soccer</h1>
+    </header>
+
+    <main>
+        <section>
+            <p>Buy <em>everything</em> you need to help your soccer star succeed.</p>
+            
+        </section>
+
+        <hr />
+        <aside>
+            <ul>
+                <li>Soccer balls and nets
+                    <ul>
+                        <li>Soccer balls</li>
+                        <li>Ball accessories</li>
+                        <li>Soccer nets</li>
+                    </ul>    
+                </li>
+                <li>Soccer shoes</li>
+                <ul>
+                    <li>Adults</li>
+                    <li>Kids</li>
+                </ul>    
```

Comment:
You should have the closing `</li>` after the `<ul>`, like you did with "Soccer balls and nets"